external/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+add_subdirectory(SPHEREPACK)
+
+# Propagate core libraries up to global scope.
+set(SPECTRE_LIBRARIES
```

Comment:
remove these last four lines (we have changed how we build the library dependencies